mitplan-frontend/src/store/mitplansSlice.ts

```diff
@@ -287,6 +287,7 @@ const mitplansSlice = createSlice({
               name: 'Sheet 1',
               assignmentEvents: {},
               encounter: getDefaultEncounter(),
```

Comment:
once you pass the props correctly to TimestampColumn, we don't need this in the state anymore